docs/spec_draft.md

```diff
@@ -167,6 +167,7 @@ described below)
    * [or](#stablehloor)
```

Comment:
I propose that we update these files now, so that reviewers look at this pull request in its fullness the entire time. 

Sure we might need a rebase before merging, but that'll be a one-time thing, right? It's not like rebases are needed every time there's an update to a pull request (the updates can be simply pushed into the PR branch). 

Overall, I feel that the benefit that we'll be getting from more thorough reviews exceeds the additional effort needed to maintain status.md and StablehloOps.td in PRs like this one.

Reply:
Sure